designer/client/components/FormDetails/__tests__/FormDetails.jest.tsx

```diff
@@ -0,0 +1,285 @@
+      });
+
+      const alphaRadio = screen.getByLabelText("Alpha") as HTMLInputElement;
+      expect(alphaRadio.checked).toEqual(false);
+
+      fireEvent.click(alphaRadio, { target: { value: "alpha" } });
+      expect(alphaRadio.checked).toEqual(true);
+
+      const saveButton = findSaveButton();
+      fireEvent.click(saveButton);
+
+      expect(providerProps.save.mock.calls[0][0]).toMatchObject({
+        name: "Default Title",
+        phaseBanner: { phase: "alpha" },
+      });
+    });
+
+    it("sets beta phase", () => {
+      renderWithDataContext(<FormDetails />, {
+        providerProps,
+      });
+
+      const noneRadio = screen.getByLabelText("None") as HTMLInputElement;
+      expect(noneRadio.checked).toEqual(true);
+
+      const betaRadio = screen.getByLabelText("Beta") as HTMLInputElement;
+      expect(betaRadio.checked).toEqual(false);
+
+      fireEvent.click(betaRadio, { target: { value: "beta" } });
+      expect(betaRadio.checked).toEqual(true);
+
+      const saveButton = findSaveButton();
+      fireEvent.click(saveButton);
+      expect(providerProps.save.mock.calls[0][0]).toMatchObject({
+        name: "Default Title",
+        phaseBanner: { phase: "beta" },
+      });
+    });
+
+    it("sets none phase", () => {
+      renderWithDataContext(<FormDetails />, {
+        providerProps: {
+          ...providerProps,
+          data: {
+            ...providerProps.data,
+            phaseBanner: { phase: "alpha" },
+          },
+        },
+      });
+
+      const alphaRadio = screen.getByLabelText("Alpha") as HTMLInputElement;
+      expect(alphaRadio.checked).toEqual(true);
+
+      const noneRadio = screen.getByLabelText("None") as HTMLInputElement;
+      expect(noneRadio.checked).toEqual(false);
+
+      fireEvent.click(noneRadio);
+      expect(noneRadio.checked).toEqual(true);
+
+      const saveButton = findSaveButton();
+      fireEvent.click(saveButton, { target: { value: "" } });
+      expect(providerProps.save.mock.calls[0][0]).toMatchObject({
+        name: "Default Title",
+        phaseBanner: { phase: undefined },
+      });
+    });
+  });
+
+  describe("Feedback form", () => {
+    it("sets `Yes` feedback form", () => {
+      renderWithDataContext(<FormDetails />, {
+        providerProps,
+      });
+
+      const yesFeedbackRadio = screen.getByLabelText("yes") as HTMLInputElement;
+      const noFeedbackRadio = screen.getByLabelText("no") as HTMLInputElement;
+
+      expect(yesFeedbackRadio.checked).toEqual(false);
+      expect(noFeedbackRadio.checked).toEqual(true);
+
+      fireEvent.click(yesFeedbackRadio);
+      expect(yesFeedbackRadio.checked).toEqual(true);
+
+      const saveButton = findSaveButton();
+      fireEvent.click(saveButton);
+      expect(providerProps.save.mock.calls[0][0]).toMatchObject({
+        name: "Default Title",
+        feedbackForm: true,
+        phaseBanner: { phase: undefined },
+      });
+    });
+
+    it("sets `No` feedback form", async () => {
```

Comment:
It seems like async has been used here without await. Is this an accident or was it done on purpose?